Projects/Metatron_Protocol_Launch/README.md

```diff
@@ -0,0 +1,69 @@
+
+## Metatron Launch and Pilot Draft Plan
+
+The *draft* plan blow provides for 3 main outcomes:
+
+1. Basic Integration of Metatron with Filecoin
+1. Launch of Metatron Protocol and Reference Implementation
+1. Configuration and Support for EDA's Pilot use of Metatron
+
+The plan currently still misses a Python software engineer with Filecoin experience. We are actively looking at finding that resource. Please contact [flaxscrip](https://flaxscrip.github.io/flaxscrip-space/) with references.
+
+![Metatron Launch and Pilot Draft Plan](Metatron_Plan.png)
+
+The plan is divided in 5 Sprints:
+1. Sprint 1: Protocol Work
+1. Sprint 2: UX Improvements and Filecoin Integration
+1. Sprint 3: Public Launch of Repo and Reference Implementation
+1. Sprint 4: EDA Pilot Configuration and Support
+1. Sprint 5: Sustaining, Postmortem, Presentation for Global Promotion
+
+Because of the scarcity of expert resources and to allow coordination time with the EDA Pilot efforts, we are proposing a plan that stretches out over 20 weeks with 2 python programmers committed to ~20 hours / week. [@macterra](https://macterra.github.io/macterra-space/) will provide architecture and technology guidance for Metatron while [@flaxscrip](https://flaxscrip.github.io/flaxscrip-space/) will manage the overall effort to completion. [aiggdev](https://github.com/aiggdev) is our main Python developer, and we are looking for a *filecoin-tbd* resource.
+
+The draft plan proposes a series of 16 deliverables that stretches out through the 20 weeks. Each deliverable can be defined as a Bounty that will be assigned to the appropriate Players. Upon completion of a deliverable, 1/16th of the project's budget would be awarded to the contributors. Each bounty can specify custom allocation of the payment to the participating players.
+
+![Metatron Project Deliverables](Metatron_Project_Deliverables.png)
+
+## Project Management and Governance
+
+There is a pre-Filecoin history to the project that can be preserved with the continued use of [EDENomicon](https://nomicon.edenprotocol.io/) as an open governance and transparent project management methodology. If a "clean slate" is desired, we could fork a new Nomicon game specifically for this launch. This will be determined after discussion with the Filecoin Foundation team.
+
+Nomicon is an open project management methodology that we have used successfully in numerous other software projects. For more information, please visit the [Software Development Game](https://nomicon.edenprotocol.io/Nomicon/) page on the EDENomicon.
+
+[![The Software Development Game](What_is_Nomicon.png)](https://nomicon.edenprotocol.io/Nomicon/SDG.pdf)
```

Comment:
Recommend removing this image/link. The Nomicon page linked above contains links to this pdf.